x/docs/md/cli/troubleshooting.md

```diff
@@ -0,0 +1,200 @@
+
+## Troubleshooting The CLI <a name="troubleshooting"></a>
+
+## Contents 
+- [mac osx: installing without homebrew](#installing-without-homebrew)
+- [windows: installing manually](#installing-manually)
+
+If you need further help: 
+- [view the general help](http://exercism.io/help) 
+- [join the exercism.io chat on gitter](https://gitter.im/exercism/support): [![Join the chat at https://gitter.im/exercism/support](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/exercism/support)
+
+
+<a name="installing-without-homebrew"></a>
+### Installing Without Homebrew 
+
+This is described in detail in [this video tutorial](https://www.youtube.com/watch?v=TCT4eHGwfaE).
+
+First you need to know which processor architecture your computer has. If
+you're not sure, you can use Terminal.app to find out:
+
+Open Terminal.app and type in the following command:
+
+<<<<<<< Updated upstream
+```bash
+=======
+```bash
+>>>>>>> Stashed changes
```

Comment:
@gusseting it looks like you committed some git conflict markers. You'll need to delete the conflict markers and the duplicate lines. https://wincent.com/wiki/Understanding_Git_conflict_markers